Dockerfile

```diff
@@ -1,3 +1,5 @@
+# syntax=docker/dockerfile:experimental
```

Comment:
This was added to fix demos, but it looks like they are working now without this change.

Reply:
No, we are adding it back, the fix was removing it.